app/jobs/find_sorns_job.rb

```diff
@@ -2,86 +2,6 @@ class FindSornsJob < ApplicationJob
   queue_as :default
 
   def perform(*args)
-    # Makes queries to the Federal Register API to find SORNs.
-    # Searching for 'Privacy Act of 1974; System of Records' of type 'Notice' is the best query we have found.
-    # In the results we still filter on those with a title that includes 'Privacy Act of 1974'.
-
-
-    conditions = { term: 'Privacy Act of 1974; System of Records' }#, agencies: ['general-services-administration'] }
-    # 'general-services-administration', 'justice-department', 'defense-department']
-    fields = ["action", "agencies", "agency_names", "citation",
-      "dates", "full_text_xml_url", "html_url", "pdf_url",
-      "publication_date", "raw_text_url", "title", "type"]
-
-    # unfortunately the ruby gem doesn't have the year filter implemented, only specific dates.
-    # we may want to start using the http api instead.
-
-    search_options = {
-      conditions: conditions,
-      type: 'NOTICE', # doesn't seem to work
-      fields: fields,
-      order: 'newest', #oldest
-      per_page: 200,
-      page: 1
-    }
-
-    search_fed_reg(search_options)
+    FederalRegisterApi.find_sorns
```

Comment:
was i under the correct impression that the FR had its own maintained API or gem? if that's the case would added something like `Client` help differentiate it?

Reply:
Yeah, any name suggestions are welcome. `Client` or `Wrapper` are better than what I have.

Reply:
Changed to FederalRegisterClient.